store/utils.go

```diff
@@ -51,3 +51,11 @@ func TenantFromDbName(dbName string, baseDb string) string {
 	}
 	return noBase
 }
+
+// DbNameForTenant composes tenant's db name.
+func DbNameForTenant(tenantId string, baseDb string) string {
```

Comment:
Can you update `DbFromContext` to use this function as well?

Reply:
fixed